tests/test_custom_checkers/test_pycodestyle_checker.py

```diff
@@ -1,406 +1,92 @@
 import os
 
 import pylint.testutils
+import pytest
 from astroid.astroid_manager import MANAGER
 
 from python_ta.checkers.pycodestyle_checker import PycodestyleChecker
 
-DIR_PATH = os.path.join(__file__, "../../../examples/custom_checkers/e9989_pycodestyle/")
+DIR_PATH = os.path.normpath(
+    os.path.join(__file__, "../../../examples/custom_checkers/e9989_pycodestyle/")
+)
+
+# Define parameter sets for different error and no-error tests
+error_params = [
+    ("e115_error.py", "E115", 2, 0, "expected an indented block (comment)"),
```

Comment:
@lana-w this is very good, but can be simplified even further: note that there's a common element: the error code (e.g. `E115`) can be used to _compute_ the filename, and so you don't need to have both in each tuple.